### ISSUE TITLE
Missing relu layer

### ISSUE THREAD

**Issue by MicheleFG** [lwtnn/lwtnn#167]
In the list of possible layers, the simple RELU layer is missing:
https://github.com/lwtnn/lwtnn/blob/master/converters/keras_v2_layer_converters.py#L156

Can it be added?

I noticed that I get different results if I add it after a BN layer rather than before it (in which case I could bypass the problem by adding it in the previous Dense layer using the "activation" attribute)

Thanks!

Michele

**Comment by dguest** [comment]
Obviously pull requests are welcome here. Failing that if you can attach the keras model you're trying to convert and more information about what is failing that will make this easier.

**Comment by MicheleFG** [comment]
I am about to go on holiday, so I don't have time to make a pull request.
The model I am trying to convert is something like this:
        input = layers.Input(shape=(10), name="Noise")
        G = layers.Dense(50)(input)
        G = layers.BatchNormalization()(G)
        G = layers.ReLU()(G)
        generator = Model(inputs=[input], outputs=G)

**Comment by MicheleFG** [comment]
I think that the function I linked above and this other line are the only places that need to be modified to add the relu layer
https://github.com/lwtnn/lwtnn/blob/master/converters/kerasfunc2json.py#L334

**Comment by chnzhangrui** [comment]
Hi here is an example input:
Run with the command:
```
kerasfunc2json.py generator_model_photons_eta_20_25_UltraLow12.json checkpoint_photons_eta_20_25_UltraLow12.h5 > variables_photons_eta_20_25_UltraLow12.json
```
[Archive.zip](https://github.com/lwtnn/lwtnn/files/9293783/Archive.zip)
